src/MfaVerifyController.js

```diff
@@ -100,6 +115,14 @@ function (Okta, Checkbox, BaseLoginController, TOTPForm, YubikeyForm, SecurityQu
           this.options.appState.get('isMfaRequired')) {
         return true;
       }
+      if (this.options.appState.get('isSuccessResponse')) {
```

Comment:
Gotcha, missed the MFA -> `PASSWORD_EXPIRED` part (still wondering why do we need this path when we already have `PRIMARY_Authentication` -> `PASSWORD_EXPIRED`...)